config/interpolate_funcs.go

```diff
@@ -23,6 +23,7 @@ func init() {
 		"element": interpolationFuncElement(),
 		"replace": interpolationFuncReplace(),
 		"split":   interpolationFuncSplit(),
+		"toint":   interpolationFuncToint(),
```

Comment:
+1 - `toint` is also only 1 character off from `taint` which is a different important concept within TF